projects/packages/publicize/src/class-publicize-utils.php

```diff
@@ -56,10 +56,6 @@ public static function should_block_editor_have_social() {
 			return false;
 		}
 
-		if ( ! self::is_publicize_active() ) {
-			return false;
-		}
-
```

Comment:
What are the implications of this? How much more code are we going to be loading and executing if we treat publicize as active?

Reply:
Script data class already uses that check to return the bare minimum data.

https://github.com/Automattic/jetpack/blob/bcaf0d5711189c79d154b750dff8232dbc91b06f/projects/packages/publicize/src/class-publicize-script-data.php#L103